content/Watsonx/WatsonxData/106.md

```diff
@@ -0,0 +1,202 @@
+
+1. Select the **Infrastructure manager** icon from the left-side menu.
+
+2. Click the **Add component** dropdown menu at the right-side of the screen. Select **Add Database**.
+
+  ![](./images/106/add-database.png)
+
+3. In the **Add database** pop-up window, select/enter the following pieces of information in the **1. Database definition** section:
+
+
+            • Database type:    IBM Db2
+            • Database name:    GOSALES
+            • Display name:     Db2DW
+            • Hostname:         192.168.252.2
+            • Port:             50000
+            • Username:         db2inst1
+            • Password:         db2inst1
+            • SSL connection:   ***Uncheck***
+
+
+  ![](./images/106/add-database-form1.png)
+
+  When a new bucket or database is added to watsonx.data, a new catalog is created as well.
+
+
+4. Scroll down to section **2. Associated catalog definition**. Enter **db2catalog** for the **Catalog name**. Then, click **Add**.
+
+  ![](./images/106/add-database-form2.png)
+
+  The **Db2DW** database and **db2catalog** catalog have been added to watsonx.data and are now reflected in the topology view of the infrastructure components.
+
+  ![](./images/106/add-database-result.png)
+
+  The **db2catalog** catalog is automatically associated with the **Db2DW** database, but to be able to query data from this database, the **db2catalog** catalog must also be associated with the **presto-01** engine.
+
+5. Hover your mouse pointer over the **db2catalog** catalog tile and the **Manage associations** icon will appear. Click the **Manage associations** icon.
+
+  ![](./images/106/manage-association.png)
+
+6. In the **Manage associations** pop-up window, select the checkbox for the **presto-01** engine and then click **Save and restart engine**.
+
+  ![](./images/106/manage-association-save.png)
+
+  A line now connects **presto-01** with **db2catalog**, indicating that they’re associated.
+
+  ![](./images/106/manage-association-connect.png)
+
+
+Next you will add the PostgreSQL database. The password for the PostgreSQL admin account is specific to your environment and the following steps will extract it for you.
+
+7. Open a terminal command window to the watsonx.data server as the root user.
+
+8. The administrative user for the PostgreSQL database is **admin**. Run the following command to extract and display the **password**. Copy the value shown to a location you can refer to later.
+
+  ```bash
+    docker exec ibm-lh-postgres printenv | grep POSTGRES_PASSWORD | sed 's/.*=//'
+  ```
+
+9. Repeat Steps 2-6 from above to add the PostgreSQL database to watsonx.data. Use the following information:
+
+            • Database type:    PostgreSQL(it can be found under Other Databases)
```

Comment:
Bullet points should be separated by newline characters so that they appear on separate lines.

Reply:
Used - from bullet points